core/settings.py

```diff
@@ -201,3 +196,23 @@
     profiles_sample_rate=1.0,
     environment=ENV or "local",
 )
+
+# Enable / Disable Azure Auth
+if os.environ.get("AZURE_AUTH_ENABLED", "true") == "true":
+    # Adds the Azure Authentication middleware to the Django Authentication middleware
+    MIDDLEWARE.append(
+        "azure_auth.middleware.AzureMiddleware",
```

Comment:
This disrupts [`"django_prometheus.middleware.PrometheusAfterMiddleware"`](https://github.com/ministryofjustice/find-moj-data/pull/342/files#diff-4392745abac424baa573e9c60413308af753eb912be194dfc224f1e71e81ef98R57), which [needs to be the final middleware entry](https://github.com/korfuri/django-prometheus?tab=readme-ov-file#quickstart)